### PR TITLE
fix(inspector2): add more efficient way to check if any active findings

### PR DESCRIPTION
**Pull request by sergargar**
### Context
For the check `inspector2_active_findings_exist` we are getting all Inspector2 findings in the service just to check if there is any active finding, which can lead to API timeouts in big AWS accounts with thousands of findings.

### Description

Only check if there is one active inspector finding in the account and in the check Prowler can still verify whether the AWS account has active inspector findings or not, only by iterating one single finding.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prowler/providers/aws/services/inspector2/inspector2_service.py [hunk 1]
```diff
@@ -34,54 +33,30 @@ def __batch_get_account_status__(self, regional_client):
...
-                                        arn=finding["findingArn"],
-                                        region=regional_client.region,
-                                        severity=finding.get("severity"),
-                                        status=finding.get("status"),
-                                        title=finding.get("title"),
-                                    )
-                                )
+            regional_client = self.regional_clients[inspector.region]
+            active_findings = regional_client.list_findings(
+                filterCriteria={
```
Comment (jfagoagas) [inline prowler/providers/aws/services/inspector2/inspector2_service.py:41]: Is this `filterCriteria` applied at server side or client side?

**Review by jfagoagas — COMMENTED** [review]
That looks great, I left some comments to clarify things.

File: prowler/providers/aws/services/inspector2/inspector2_service.py (hunk 1 above)
Comment (sergargar) [reply prowler/providers/aws/services/inspector2/inspector2_service.py:41]: Server side, that is why is way faster now.

**Review by jfagoagas — APPROVED** [review]
Great catch @sergargar thanks for this!

**Review by jfagoagas — APPROVED** [review]

**Pull request merged**